tools/Config.mk

```diff
@@ -363,3 +363,9 @@ define TESTANDREPLACEFILE
 	fi
 endef
 endif
+
+# Some defaults.
+# $(TOPDIR)/Make.defs can override these appropriately.
+
+MODULECC = $(CC)
```

Comment:
Should we select a better name? because elf binary rule also need to use macro.

Reply:
i'd suggest

* MODULECC for modules
* ELFCC for ELF binaries

to align with the existing CMODULEFLAGS/CELFFLAGS

Reply:
Ok. Change to:
MODULECC ?= $(CC)?